src/components/tool/guide/Config/ConfigPicker.js

```diff
@@ -0,0 +1,78 @@
+import { CheckCircleIcon, WarningIcon } from "@chakra-ui/icons";
+import {
+  Alert,
+  AlertIcon,
+  Box,
+  CloseButton,
+  Flex,
+  Heading,
+  Tooltip,
+} from "@chakra-ui/react";
+
+import { useState } from "react";
+
+import InputPicker from "../../../input/InputPicker";
+
+export default function ConfigPicker({
+  title,
+  description,
+  selected,
+  inputProps,
+  onChange = {},
+  showAlert = false,
+  alertDismissible = true,
+  alertType = "info",
+  alertMsg = null,
+}) {
+  const [alertDismissed, setAlertDismissed] = useState(false);
+  const onClose = () => {
+    setAlertDismissed(true);
+  };
+
+  return (
+    <>
+      <>
+        <Box>
+          {selected ? (
```

Comment:
Can we group the logic for the headings and alerts into components that are used in `ConfigSlider`, `ConfigPicker`, and `ConfigMultipleNumbers`?

Reply:
Updated